inst/components/scss/grid.scss

```diff
@@ -3,13 +3,19 @@
   gap: var(--bslib-spacer, 1rem);
   height: var(--bslib-grid-height);
   &.grid {
+    // Don't let intrinsic width of a column affect the width of grid cols
+    grid-template-columns: repeat(var(--bs-columns, 12), minmax(0, 1fr));
     // For some reason, Bootstrap sets `grid-template-rows: 1fr` by default, which
     // is problematic for a multi-row filling layout. This fixes it...
     // > page_fillable(layout_columns(c(12, 12), plotly::plot_ly(), plotly::plot_ly()))
     grid-template-rows: unset;
     grid-auto-rows: var(--bslib-grid--row-heights);
     @include bslib-breakpoints-css-vars('bslib-grid--row-heights', map-keys($grid-breakpoints));
   }
+
+  & > * > .shiny-input-container {
```

Comment:
Is the extra `> *` needed here because of the `layout_columns(fillable = T)` thing?

Reply:
Yeah `layout_columns()` adds a doc with the bs grid classes and `layout_column_wrap()` puts children in a fillable container. I know the first is always there but I should double check if the second is dependent on the value of `fillable`

Reply:
Oh yeah, they both wrap their children in a `grid_item_container()`